perf/security/sds-tests/egress-sds/README.md

```diff
@@ -0,0 +1,50 @@
+# Test description
+
+This is a TLS egress gateway SDS test.
+The test sets up a number of TLS egress gateways for a group of nginx services.
+The test creates a group of sleep pods, where each sleep sends HTTP requests to an nginx
+service periodically. For example, sleep-1 sends HTTP requests to my-nginx-1,
+and sleep-2 sends HTTPS requests to my-nginx-2. The sleep pods are deployed in clientns namespace and nginx services are deployed
```

Comment:
if `clientns` is the name of your namespace, please add quotes to it to differentiate from the normal text.